Directory.Packages.props

```diff
@@ -21,16 +21,16 @@
     <PackageVersion Include="NLog" Version="5.2.8" />
     <PackageVersion Include="NUnit.Analyzers" Version="4.0.1" />
     <PackageVersion Include="SecurityCodeScan.VS2019" Version="5.6.7" />
-    <PackageVersion Include="System.IO.Abstractions" Version="20.0.15" />
+    <PackageVersion Include="System.IO.Abstractions" Version="20.0.28" />
     <PackageVersion Include="System.IO.FileSystem" Version="4.3.0" />
     <PackageVersion Include="Terminal.Gui" Version="1.15.1" />
-    <PackageVersion Include="Tesseract" Version="4.1.1" />
+    <PackageVersion Include="Tesseract" Version="5.2.0" />
```

Comment:
Tesseract 5 has some significant changes from 4 so we'd been staying on 4 for now